app/assets/javascripts/components/sidebar/post_component.js.coffee

```diff
@@ -0,0 +1,49 @@
+window.PostComponent = React.createClass
```

Comment:
this component should have a `propTypes` where you specify that `post` and `project` are required props and document their shape. This is both nice from a self-documenting-in-code perspective as well as it enforces the constraints of which props are required, which it seems to me are `post` and `project`, and also `user` I believe.

Example:
`propTypes: {
  user: React.PropTypes.object.isRequired,
  project: React.PropTypes.shape({
    name: React.PropTypes.string.isRequired
  }),
  post: React.PropTypes.shape({
    num_comments: React.PropTypes.number.isRequired,
    text: React.PropTypes.string.isRequired
  })
}`